docs/modules/pipelines/pages/job-monitoring.adoc

```diff
@@ -0,0 +1,34 @@
+= Monitoring Jobs
+:description: To ensure the reliability of streaming jobs and their workloads, try implementing some basic monitoring. With Hazelcast, you can monitor the status of streaming jobs to quickly identify job failures for troubleshooting.
+
+{description}
+
+== Monitoring for Status Changes
+
+There are two ways to monitor for job status changes. You can either poll the `Job.getStatus()` method separately for each job or add a `JobStatusListener` to your jobs.
+
+== Using a JobStatusListener
+
+The `JobStatusListener` has several advantages over polling for the job status. 
+
+- You can write custom code for execution on status change. For example, triggering an alert or writing the changed status to a dashboard.
+
+- You can also identify jobs in an error state. By default, xref:troubleshoot:error-handling.adoc#processing-guarantees[all streaming jobs are suspended on failure]. When a job moves to the suspended status, the listener is passed a suspension cause; either an error or user canceled. Use the suspension cause to filter out all jobs canceled by a user. 
+
+NOTE: The suspension cause is also displayed on the Jobs page in Management Center.
```

Comment:
MC will hopefully show also `isUserCancelled` flag

Reply:
I've reworded this to suspension reason so that an error or user action are covered.